### DISCUSSION TITLE
swanctl.conf %any with remote_addrs

### DISCUSSION THREAD

**Discussion by anav000** [strongswan/strongswan#2080] [Q&A] [answered]
Hi there,

I wanted to know if there is a value similar to "%any" for the keyword "remote_addrs" please ? 
(as it existed with ipsec.conf)

In my case, I want to be flexible with remote addresses.

Thank you for your help!

K.R.

**Comment by 3157696** [comment]
Can't

**Comment by tobiasbrunner** [comment, marked as answer]
Sure, works the same way.
